History.md

```diff
@@ -42,6 +39,12 @@
 
   [Feature Request #194](https://github.com/meteor/meteor-feature-requests/issues/194)
 
+* To support developers running on a 32-bit OS, Meteor now supports both 32
```

Comment:
Put a hyphen after 32: "both 32- and 64-bit versions"

Reply:
Am I copy-editing English prose because PowerShell is so foreign to me? Perhaps.

Reply:
[PowerShell approved verbs](https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx) are prose, of sorts. 🤕 

Mind you, I only followed them loosely.